### ISSUE TITLE
Option to hide version number and issues link from status bar

### ISSUE THREAD

**Issue by fbnlsr** [HvyIndustries/crane#191]
I think it would be nice to have an option that would allow users to hide Crane's version number and issues link from the status bar. Although these info can have some use to some, they clutter the status bar for most and should be optional.

**Comment by nevadascout** [comment]
Thanks for the suggestion!

We can definitely add an option to hide the version number, but you can already hide the issues link in the status bar by setting `crane.showStatusBarBugReportLink` to `false` in your VS Code user settings.

**Comment by fbnlsr** [comment]
Thanks @nevadascout. I think having Crane report if it's currently indexing or if it's done indexing is valuable information though. Once I set `crane.showStatusBarBugReportLink` to `false` and restarted Code, I had no feedback why Code was suddenly unresponsive. It was because Crane was indexing the files I was working on.

Here's a suggestion: Crane should report when it's indexing files and when it's done indexing them in the status bar. Once the indexing is done, the message should disappear after x seconds (TBD). A few options should let the user:

- show/hide Crane's version number
- show/hide Crane's "Found a bug?" link
- show/hide Crane's indexing status

**Comment by nevadascout** [comment]
Crane v0.3.0 removes the version number indicator and fixes the issue hiding the indexing status when the bug report link is disabled.

**Comment by gnowland** [comment]
Version indicator still appearing in crane v0.3.8 😕 please add a setting to allow it to be hidden 
<img width="97" alt="screen shot 2017-08-14 at 12 47 56 pm" src="https://user-images.githubusercontent.com/4430119/29288817-10657a16-80ef-11e7-8f35-ef14b17ee49e.png">

**Comment by nevadascout** [comment]
@gnowland I think you can set the `crane.showStatusBarBugReportLink` setting to false and it'll disappear (I can't remember though)

Let me know if it works? If not I'll add a setting to disable it properly!

**Comment by gnowland** [comment]
@nevadascout `crane.showStatusBarBugReportLink` doesn't remove the version indicator, but it does remove the bug report link. I think the version indicator *may* possibly be supposed to disappear after x seconds but it doesn't do that for me, at least not consistently...

**Comment by nevadascout** [comment]
@gnowland Okay thanks for letting me know, I'll add a setting to disable it in the next version!

EDIT: Created #357 for this
